service/dap/types.go

```diff
@@ -208,10 +211,10 @@ func (m *SubstitutePath) UnmarshalJSON(data []byte) error {
 		}
 		return err
 	}
-	if tmp.From == "" || tmp.To == "" {
+	if tmp.From == nil || tmp.To == nil {
```

Comment:
This isn't very useful for the cli, it can already be run from a different directory and will use relative paths.

Reply:
Yes, the main use case is to achieve relative path support for delve dap using `launch.json`. 

rules specified via the DEBUG CONSOLE should in theory work in the same way but I don't know how empty string can be passed without delimiters.
```
dlv config substitutePath /prefix/  ""     // This will delete the `/prefix/` mapping only.
```

As @aarzilli mentioned, in CLI this isn't a problem considering the binary can be already run in a different directory using `--wd`

Reply:
Added new commit to support for `dlv config substitutePath`.
```
dlv config substitutePath /a/path ""  // Used to be equal doing dlv config substitutePath /a/path. Now it's updating rule with empty string
dlv config substitutePath "" /a/path // Was already working.
dlv config substitutePath "" // Used be error always. Now it's an error when there's no mapping to empty string. Otherwise deletes empty string mapping
```